examples/state.redis/go.mod

```diff
@@ -6,14 +6,14 @@ replace github.com/dapr-sandbox/components-go-sdk => ../../.
 
 require (
 	github.com/dapr-sandbox/components-go-sdk v0.0.0-00010101000000-000000000000
-	github.com/dapr/components-contrib v1.8.0-rc.1.0.20220924063709-b0d267317d47
+	github.com/dapr/components-contrib v1.9.0-rc.1
```

Comment:
This will prob need to be updated.

Reply:
done!

Reply:
actually we are pointing to 1.9.0-rc.5 on dapr tag v1.9.0 https://github.com/dapr/dapr/blob/v1.9.0/go.mod#L11